orbit/pkg/update/notifications.go

```diff
@@ -501,14 +501,14 @@ func (w *windowsMDMBitlockerConfigFetcher) attemptBitlockerEncryption(notifs fle
 		ClientError:   bitlockerError,
 	}
 
+	errServerUpdate := w.EncryptionResult.SetOrUpdateDiskEncryptionKey(payload)
 	if err != nil {
```

Comment:
I think here you want this?
```suggestion
	if errServerUpdate != nil {
```